src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionAzureKeyVaultProvider.java

```diff
@@ -667,7 +667,7 @@ public boolean verifyColumnMasterKeyMetadata(String masterKeyPath, boolean allow
         boolean append = true;
         if (null != mssqlJdbcProperties) {
             String endpoints = mssqlJdbcProperties.getProperty(AKV_TRUSTED_ENDPOINTS_KEYWORD);
-            if (null != endpoints && !endpoints.isBlank()) {
+            if (null != endpoints && !endpoints.trim().isEmpty()) {
```

Comment:
`endpoints.trim().length() > 0` would be more performant than an `isEmpty()` check.

Reply:
more readable and minimal difference